src/component/Header/Header.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface HeaderProps {
+  todos: Todo[],
+  addTodo: () => void,
```

Comment:
The addTodo function should accept a string parameter for the todo title. Change the type definition of addTodo to: (title: string) => void